components/scream/docs/build.md

```diff
@@ -69,13 +69,13 @@ usually looks something like the following:
 cd $RUN_ROOT_DIR
 cmake \
     -D CMAKE_BUILD_TYPE=Debug \
-    -D KOKKOS_ENABLE_DEBUG=TRUE \
-    -D KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION=OFF \
-    -D KOKKOS_ENABLE_SERIAL=ON \
-    -D KOKKOS_ENABLE_OPENMP=ON \
-    -D KOKKOS_ENABLE_PROFILING=OFF \
-    -D KOKKOS_ENABLE_DEPRECATED_CODE=OFF \
-    -D KOKKOS_ENABLE_EXPLICIT_INSTANTIATION:BOOL=OFF \
+    -D Kokkos_ENABLE_DEBUG=TRUE \
+    -D Kokkos_ENABLE_AGGRESSIVE_VECTORIZATION=OFF \
+    -D Kokkos_ENABLE_SERIAL=ON \
+    -D Kokkos_ENABLE_OPENMP=ON \
+    -D Kokkos_ENABLE_PROFILING=OFF \
+    -D Kokkos_ENABLE_DEPRECATED_CODE=OFF \
+    -D Kokkos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=OFF \
```

Comment:
This directive is no longer recognized by cmake. It appears the new directive is `KOKKOS_ENABLE_ETI`. I don't know why they changed it, nor why they use KOKKOS instead of Kokkos for this one, unlike every other cmake option.

Reply:
Thanks for the heads-up, @bartgol .